packages/flutter/lib/src/widgets/app.dart

```diff
@@ -185,13 +189,13 @@ class WidgetsApp extends StatefulWidget {
 
 class _WidgetsAppState extends State<WidgetsApp> implements WidgetsBindingObserver {
   GlobalObjectKey<NavigatorState> _navigator;
-  LocaleQueryData _localeData;
+  Locale _locale;
```

Comment:
i guess it would mean having to have a didUpdateWidget method and so on, which isn't great